src/components/DataCollector/validateUserEventOptions.js

```diff
@@ -27,6 +27,10 @@ export default ({ options }) => {
     data: objectOf({}),
     renderDecisions: boolean(),
     decisionScopes: arrayOf(string()).uniqueItems(),
```

Comment:
I am wondering if we shouldn't make `decisionScopes` and `personalization` mutually exclusive

Reply:
I don't think we should do that, since we want to maintain backwards compatibility at least initially. If we do decide to enforce mutual exclusivity, it should be done as part of a future task for deprecating the top-level `decisionScopes` option.